code/modules/food_and_drink/plants.dm

```diff
@@ -629,8 +575,7 @@ ABSTRACT_TYPE(/obj/item/reagent/containers/food/snacks/plant)
 
 	make_reagents()
 		..()
-		var/datum/plantgenes/DNA = src.plantgenes
-		reagents.add_reagent("cryostylane", DNA.potency)
+		reagents.add_reagent("cryostylane", 10)
```

Comment:
what's with this potency change?

Reply:
To be honest, in hindsight, it should have been a different PR. I originally wanted to go over most produce types and get rid of any chems that get added by this proc when they are harvested.

Reply:
Reverted the changes done to it. Will most likely put changes like that in its own PR.